rmgpy/statmech/ndTorsions.py

```diff
@@ -597,12 +598,22 @@ def calc_partition_function(self, T):
         """
         calculate the classical/semiclassical partition function at a given temperature
         """
-        rngs = [(0.0, 2.0 * np.pi) for x in range(len(self.pivots))]
 
         def f(*phis):
-            return self.rootD(*phis) * np.exp(-self.V(*phis) / (constants.R * T))
-
-        intg = inte.nquad(f, ranges=rngs)[0]
+            return (self.rootD(*phis)) * np.exp(-self.V(*phis) / (constants.R * T))
+        
+        rphis = np.linspace(0, 2.0*np.pi, 30)
```

Comment:
I never seen anyone write *args operators that way and I think the spacing would make it more easy to confuse with multiplication with a missing first argument. Since it's an operation on the variable itself I think it makes the most sense to not have a space.